node/Makefile

```diff
@@ -345,6 +345,10 @@ test_image: .calico_test.created
 calico-node.tar: $(NODE_CONTAINER_CREATED)
 	docker save --output $@ $(NODE_IMAGE):latest-$(ARCH)
 
+calico-typha.tar: ../go.mod $(shell find ../apiserver -name '*.go') $(shell find ../libcalico-go -name '*.go')
```

Comment:
should be `typha` instead of `apiserver` here

Reply:
ah, right. fixed it.